examples/tv-app/android/java/AppImpl.cpp

```diff
@@ -374,6 +331,18 @@ EndpointId ContentAppFactoryImpl::AddContentApp(ContentAppImpl * app, jobject co
     return epId;
 }
 
+EndpointId ContentAppFactoryImpl::AddContentApp(ContentAppImpl * app, EndpointId desiredEndpointId)
+{
+    DataVersion dataVersionBuf[ArraySize(contentAppClusters)];
```

Comment:
Is this ok to be stored on stack? I am trying to trace the code and in src/app/util/attribute-storage.cpp it looks like data is preserved:

```
emAfEndpoints[index].dataVersions   = dataVersionStorage.data()
```

which seems that this buffer will be used persistently. Please explain data lifetime here.

Reply:
This needs resolution - I believe current code has a use-after-free (not in ram but on stack)

Reply:
Made change to allocate on stack and deallocate when removing endpoint.